lighthouse-core/report/html/renderer/i18n.js

```diff
@@ -44,9 +44,19 @@ class I18n {
    * @param {number=} granularity Controls how coarse the displayed value is, defaults to 0.1
    * @return {string}
    */
-  formatBytesToKB(size, granularity = 0.1) {
+  formatBytesToKiB(size, granularity = 0.1) {
     const kbs = this._numberFormatter.format(Math.round(size / 1024 / granularity) * granularity);
-    return `${kbs}${NBSP2}KB`;
+    return `${kbs}${NBSP2}KiB`;
```

Comment:
@connorjclark and I talked about this.

This needs to be i18n'd in the future. Half our "KiB" are translated, and half aren't. We should be consistent and allow these to be translated as well.

This problem exists today, so not blocking for this PR, but it's a TODO.